packages/components/src/components/tag/tag-utils.ts

```diff
@@ -40,6 +37,7 @@ export const getThemedBackgroundHoverColor = (
   const colorMap: Record<Exclude<TagColor, TagColorDeprecated>, string> = {
     'background-base': themedColors[`backgroundColor${keySuffix}`],
     'background-surface': themedColors[`backgroundSurfaceColor${keySuffix}`],
+    'background-frosted': changeColor(themedColors.backgroundFrostedColor, isDark ? 10 : -10),
```

Comment:
If we want to use the same hover color for all components maybe specify this somewhere more globally?

Reply:
I've abstracted it as lighten / darken function with fixed lightness factor for now.